common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromAuthority( final AzureActiveDirectoryAuthority authority) {
```

Comment:
Javadoc on all public methods

Reply:
[7453c3b](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2495/commits/7453c3bf9cbe16bbc888a59094413c0fb2872d26)